### PR TITLE
feature: added wishlist page

### PR DESCRIPTION
**Pull request by aliamaanansari**
https://neocart.netlify.app/wishlist/

### CODE REVIEW

File: wishlist/styles.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  line-height: 24px;
+}
+.product-price {
+  text-align: center;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 36px;
+  color: #000000;
+}
+.cart-btn {
```
Comment (poojakelkar) [inline wishlist/styles.css:80]: Add some padding to the button

File: wishlist/styles.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 40px;
+  color: #000000;
+  text-align: center;
+}
+.products-container {
+  padding: 2.5rem 5rem;
+  background: #f7f7f7;
```
Comment (poojakelkar) [inline wishlist/styles.css:11]: add root element for colors

**Review by poojakelkar — COMMENTED** [review]
added some changes, otherwise looks nice to merge

File: wishlist/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    />
+    <link
+      rel="stylesheet"
+      href="https://keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/styles/utility.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
```
Comment (shuntzhere) [inline wishlist/index.html:25]: It's not clear why are you importing so many sheets. You can make one main and import everything there and use that across.

File: wishlist/index.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <title>E-commerce</title>
+  </head>
+  <body>
+     <nav class="navbar">
+      <a href="/" class="navbar-title">Neo Cart</a>
+      <div class="navbar-search-input-container">
+        <div class="input-lefticon">
+          <img src="/assets/images/search-icon.svg" alt="" />
```
Comment (shuntzhere) [inline wishlist/index.html:33]: Do give an alt name.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Have suggested some changes, do give them a look.

**Pull request merged**